### PR TITLE
Add community members page

### PR DESCRIPTION
**Pull request by svrnm**
This PR adds a page under /community that shows all members in their roles across the OpenTelemetry project. It also includes a script to update this data, if we want to add this to the page, I can turn that into a scheduled workflow. 

Preview: https://deploy-preview-4970--opentelemetry.netlify.app/community/members/

cc: @open-telemetry/sig-contributor-experience-approvers

### CODE REVIEW

**Comment by theletterf** [comment]
Wow! I love it!

Any chance we'd split members by project using headings?

**Comment by svrnm** [comment]
> Wow! I love it!
> 
> Any chance we'd split members by project using headings?

I purposfully didn't do that, because I wanted to have every person only represented once (except GC/TC), and also not make things to complicated. The per-subproject info is available on their repositories. What I was thinking about as a follow up feature is that there is a filter button and you can change that few to only show you maintainers by SIG

**Review by mx-psi — COMMENTED** [review]
I think this is a great idea! The current CSS styling for the tables doesn't work well on small screens though, this is how it looks on a Pixel 7 screen 
image

(screenshot from devtools but it looks the same as in my personal device)

**Comment by svrnm** [comment]
@mx-psi thanks! I just applied some grid options from bootstrap to address that:

```
<div class="col-6 col-md-4 col-lg-3 col-xxl-2">
```

**Comment by mx-psi** [comment]
You can see the images now :) 
image

There are a couple of profile pictures that seem to be lower resolution and look odd, maybe we can enforce the same size for all profile pictures as well?

**Comment by codefromthecrypt** [comment]
I like the progress here, might make the icons a bit smaller but no need to action that.

Is there any way to have text of the github ID here without making it messy? Right now I found myself hovering over each image ;)

File: .cspell.yml
Comment (chalin) [inline .cspell.yml:11]:
```suggestion
  - data/community/members.yaml
```

File: gulp-src/lint-md.js
Comment (chalin) [inline gulp-src/lint-md.js:17]: This is fine as is. I'm curious if we could get away with `!**/node_modules/**`.

**Review by chalin — APPROVED** [review]
Wow, nice work! ✨  RSLGTM

**Review by danielgblanco — APPROVED** [review]
Love it!

File: gulp-src/lint-md.js
Comment (svrnm) [reply gulp-src/lint-md.js]: We can, fixed it :-)

**Review by mx-psi — APPROVED** [review]

**Comment by svrnm** [comment]
> There are a couple of profile pictures that seem to be lower resolution and look odd, maybe we can enforce the same size for all profile pictures as well?

The flexible layout provided by bootstrap makes this not a trivial task, but I also do not want to write something on my own. It's not feature perfect, so I would suggest we iterate on it later.

> Is there any way to have text of the github ID here without making it messy? Right now I found myself hovering over each image ;)

done

**Review by codefromthecrypt — APPROVED** [review]
:shipit: thanks a lot!

**Comment by svrnm** [comment]
FYI, I want to present this to the maintainers first (via slack/maintainers call) before merging to get a broad feedback for that.

File: data/community/members.yaml
Comment (cijothomas) [inline data/community/members.yaml:2]: could we show name along with github alias?

Comment (svrnm) [reply data/community/members.yaml:2]: I would guess so, the name should be accessible via the github API

**Comment by svrnm** [comment]
Note, that I will need to remove the images from this PR, the images are sourced from github and we do not have full control what does images show and represent, so we have a risk of showing something on the website we don't want to have there.

**Comment by codefromthecrypt** [comment]
> we have a risk of showing something on the website we don't want to have there.

honestly I did think about this, too, and wondered how we might deal with it

**Comment by svrnm** [comment]
> > we have a risk of showing something on the website we don't want to have there.
> 
> honestly I did think about this, too, and wondered how we might deal with it

Not showing them is the only viable option. Another one would pulling and storing the images locally, and then reviewing them every time we update the list but that sounds like too much effort for me.

Note, that with the latest commit I fixed that and turned the lists in to html unordered lists and added tags per user with the SIGs

File: data/community/members.yaml
Comment (svrnm) [reply data/community/members.yaml:2]: from what I understand unfortunately that would need another API call per user, I will leave that out for now and use the github handles

File: data/community/members.yaml
Comment (cijothomas) [inline data/community/members.yaml:51]: Looks like TC list is duplicated into spec-sponsors. Is that the intend? I think its best to remove TC from spec-sponsor list.

Comment (svrnm) [reply data/community/members.yaml:51]: good catch! Let me fix that

**Pull request merged**
